paddle/fluid/operators/controlflow/conditional_block_op_helper.cc

```diff
@@ -31,7 +31,10 @@ static bool IsMatchedConditionalBlockOpAndConditionalBlockGradOp(
 static void FindAllConditionalBlockAndConditionalBlockGradOp(
     const framework::ProgramDesc &program, std::vector<OpVariant> *fwd_ops,
     std::vector<OpVariant> *bwd_ops) {
-  PADDLE_ENFORCE_GE(fwd_ops->size(), bwd_ops->size());
+  PADDLE_ENFORCE_GE(
+      fwd_ops->size(), bwd_ops->size(),
+      platform::errors::InvalidArgument(
+          "Size of forward ops must be greater or equal to backward ops"));
```

Comment:
May be better to print `fwd_ops->size()` and `fwd_ops->size()`.

Reply:
Done